flashinfer/jit/cubin_loader.py

```
@@ -136,18 +136,16 @@ def download_file(
-    with open(local_path, "r") as f:
-        for line in f:
-            sha256, filename = line.strip().split()
-            if ".h" in filename:
-                return sha256
-    raise ValueError(f"Invalid path: checksums.txt not found in {checksum_path}")
+    checksums_lines = checksums_bytes.decode("utf-8").splitlines()
+    for line in checksums_lines:
+        sha256, filename = line.strip().split()
+        if ".h" in filename:
+            return sha256
+    raise ValueError("Invalid checksums.txt, no flashinferMetaInfo.h found")
```

Ruff (0.14.1)

Avoid specifying long messages outside the exception class

(TRY003)